DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/CourseDelegates/Index.cshtml

```diff
@@ -33,19 +33,11 @@
              asp-route-sortDirection="@Model.CourseDetails?.SortDirection"
              asp-route-existingFilterString="@Model.CourseDetails?.ExistingFilterString"
              role="button">
-            Export current
+            Export
           </a>
         </div>
       </div>
 
-      <div class="nhsuk-grid-row">
-        <div class="nhsuk-grid-column-full">
-          <p>
-            Choose a course to see delegates enrolled on it. If you'd like to export delegates on all courses, click "Export all" above.
-          </p>
-        </div>
-      </div>

```

Comment:
I'm not sure this was what Olivia meant, only to remove "If you'd like to export delegates on all courses, click "Export all" above."